resources/variants/mingda_magician_max_0.2.inst.cfg

```diff
@@ -0,0 +1,12 @@
+[general]
+name = 0.2mm Nozzle
+version = 4
+definition = mingda_MagicianMax
```

Comment:
Thanks! However the tests are now failing because you forgot to update the references to those files in places like this!

Reply:
Sorry, I'll improve it right away

Reply:
@Hi Ghostkeeper, I found that the checks have failed but I don't know what else is wrong.  Could you help me?

Reply:
Sorry for the confusion. The failing tests are not your fault. We updated to Python 3.10 and made some changes to the build environment for that. But the CI environment is not yet updated.

You can always run the tests locally if you have checked out the source code and have CMake and Pytest installed, by running `make test`. Or for just the unit tests of the main source code, `ctest --verbose -R pytest-main`